Dockerfile

```
@@ -1,7 +1,11 @@
 FROM eclipse-temurin:17 AS build
 
 WORKDIR /app
-COPY . .
+COPY ./src ./src
+COPY ./gradle ./gradle
+COPY ./build.gradle.kts ./build.gradle.kts
+COPY ./settings.gradle.kts ./settings.gradle.kts
+COPY ./gradlew ./gradlew
```

I think this would work:
```suggestion
COPY --exclude=entrypoint.sh  . .
```

---

ack, I'll try (but I feel like the exclude pattern will be the same order of magnitude as the include one). I believe I'll move the "optimize" part in a distinct PR so we don't tie it to the entrypoint change (I hesitated to do so at the beginning, I should have done).

---

See #456